projects/packages/forms/src/contact-form/class-form-response.php

```diff
@@ -0,0 +1,806 @@
+<?php
+/**
+ * Form_Response class.
+ *
+ * @package automattic/jetpack-forms
+ */
+
+namespace Automattic\Jetpack\Forms\ContactForm;
+
+use WP_Post;
+/**
+ * Handles the response for a contact form submission.
+ *
+ * Form_Response objects are there to help us interact with the form response data.
+ */
+class Form_Response {
+
+	const POST_TYPE = 'feedback';
+
+	/**
+	 * The Post Object of the feedback entry.
+	 *
+	 * @var WP_Post|null
+	 */
+	protected $feedback_post;
+
+	/**
+	 * The form field values.
+	 *
+	 * @var array
+	 */
+	protected $fields = array();
+
+	/**
+	 * The original Contact_Form object, if available.
+	 *
+	 * @var Contact_Form|null
+	 */
+	protected $form;
+
+	/**
+	 * The current post object, if available.
+	 *
+	 * Usallyt this is accostied with the global $post object.
+	 *
+	 * @var WP_Post|null
+	 */
+	protected $current_post;
+
+	/**
+	 * The current page number associated with the feedback entry.
+	 *
+	 * This is used to track the page number of the form that the feedback was submitted from.
+	 *
+	 * Usally this is accosited with the global $page number varieble.
+	 *
+	 * @var int
+	 */
+	protected $current_page_number = 1;
+
+	/**
+	 * The status of the feedback entry.
+	 *
+	 * @var string
+	 */
+	protected $status = 'publish'; // Default status is 'publish' or other statuses as needed.
+
+	/**
+	 * The IP address of the user who submitted the feedback.
+	 *
+	 * This is only available on form submissions, and might not be available when retrieving existing feedback posts in case the site admin decides to not store the IP address.
+	 *
+	 * @var string|null
+	 */
+	protected $ip_address = null;
+
+	/**
+	 * The subject of the feedback entry.
+	 *
+	 * @var string
+	 */
+	protected $subject = '';
```

Comment:
what the subject would be?

Reply:
This is used by akismet. To know if things are spam or not